### ISSUE TITLE
[Bruno Baptista] Submission of Committer Quote

### ISSUE THREAD

**Issue by brunobat** [microprofile/microprofile-marketing#349]
Testimonial: 
Committer since 2019, I've contributed to the project, did presentations about Microprofile and actively use almost all the existing  specs in my production projects. I believe Microprofile is the best thing that happened to enterprise Java in a very long time. I value the collaboration, the community building and having fun with the great people working on it.

Name: Bruno Baptista
Title: Technical Advisor (Software Architect)
Twitter: @brunobat_
Bio/blog: https://brunobat.com
Github: @brunobat 
Photo: https://avatars2.githubusercontent.com/u/2318030?s=460&u=123eafcbc5eb3bcf00fa21336df9d55f74c2e859&v=4

**Comment by aeiras** [comment]
Nice quote Bruno!!

I have adjusted a few things to enable your review:

_A MicroProfile Committer since 2019, I've contributed to MicroProfile with presentations and actively use almost all the existing specifications in my production projects. I believe MicroProfile is the best thing that happened to the Enterprise Java ecosystem in a very long time. I value its open-collaboration, community building while having fun with the great people working on it._

Adjust at will or decline to accept adjustments as well. Once the quote review is completed, it will be added to the [excel sheet](https://docs.google.com/spreadsheets/d/1nsNCrv9NCRVDm9BDxQELnIZair3d75LJsyEAl9cGAJs/edit#gid=0) for processing :)
